rpc/errors.go

```diff
@@ -95,3 +100,15 @@ func (e *shutdownError) Code() int {
 func (e *shutdownError) Error() string {
 	return "server is shutting down"
 }
+
+// connectionClosedError is thrown when the connection is closed
+type connectionClosedError struct {
+}
+
+func (e *connectionClosedError) Code() int {
```

Comment:
Where is this code used? If it's needed to satisfy some interface somewhere, it would be nice to add it as a doc.